lib/email-validation/strategy.rb

```diff
@@ -17,11 +17,10 @@ def verify_email(email)
         end
       end
 
-      validation_result ||= ValidationResult.new(true, true)
-
-      BlacklistedEmail.create(email: email, origin: validation_result.reason) unless validation_result.valid?
-
-      msg = validation_result.valid? ? '' : EmailValidation::incorrect_email_message(email)
+      unless validation_result.valid?
```

Comment:
Oh, ok, you probably meant https://github.com/bbatsov/ruby-style-guide#unless-for-negatives :) I'm not convinced, but let's do it your way then